src/nft/ZoraCreator1155Impl.sol

```diff
@@ -248,17 +259,40 @@ contract ZoraCreator1155Impl is
     /// @param newURI The URI for the token
     /// @param maxSupply The maximum supply of the token
     function setupNewToken(
-        string memory newURI,
+        string calldata newURI,
         uint256 maxSupply
     ) public onlyAdminOrRole(CONTRACT_BASE_ID, PERMISSION_BIT_MINTER) nonReentrant returns (uint256) {
+        uint256 tokenId = _setupNewTokenAndPermission(newURI, maxSupply, msg.sender, PERMISSION_BIT_ADMIN);
+
+        return tokenId;
+    }
+
+    /// @notice Set up a new token with a create referral
```

Comment:
could you please explain in this comment what a create referral is and the effect is has?